packages/frontend/src/view/components/BlockchainAddress.tsx

```diff
@@ -48,28 +50,39 @@ export function BlockchainAddress(props: Props) {
 
       {addressInfo && !props.full ? (
         <>
-          <Tooltip text={props.address.toString()} className="md:hidden">
-            <a href={explorerUrl} target="_blank" className="underline">
-              {addressEllipsis(props.address)}
-            </a>{' '}
-          </Tooltip>
-          <Tooltip text={'Show on ' + explorerName} className="hidden md:block">
-            <a href={explorerUrl} target="_blank" className="underline">
-              {props.address.toString()}
-            </a>{' '}
-          </Tooltip>{' '}
+          <span className={props.className}>
+            <Tooltip text={props.address.toString()} className="md:hidden">
+              <a href={explorerUrl} target="_blank" className="underline">
+                {addressEllipsis(props.address)}
+              </a>
+            </Tooltip>
+            <Tooltip
+              text={'Show on ' + explorerName}
+              className="hidden md:block"
+            >
+              <a href={explorerUrl} target="_blank" className="underline">
+                {props.address.toString()}
+              </a>
+            </Tooltip>
+          </span>
           {addressInfo.name.length > 0 && (
-            <span className="whitespace-nowrap text-xs text-zinc-500">
-              ({addressInfo.name})
-            </span>
+            <>
+              {' '}
+              <span className="whitespace-nowrap text-xs text-zinc-500">
+                {addressInfo.name}
```

Comment:
Do we want to remove brackets around the verified contract name?

Reply:
<img width="300" alt="image" src="https://github.com/l2beat/lz-monitoring/assets/67391475/1786299a-4b8a-4ea0-97dc-598e093180a2">

Reply:
[4a13884](https://github.com/l2beat/lz-monitoring/pull/97/commits/4a138840cda40c5eba275be585ddc8debd6e959a)